.github/workflows/triage_issues.yml

```
@@ -8,7 +8,7 @@ permissions:
 
 jobs:
   add_label:
-    if: github.event.action == 'opened'
+    if: github.event.action == 'opened' && secrets.ADD_TO_PROJECT_PAT != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)